eng/WpfArcadeSdk/tools/ApiCompat.targets

```diff
@@ -0,0 +1,67 @@
+<Project>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.DotNet.ApiCompat" Version="$(MicrosoftDotNetApiCompatVersion)"/>
+  </ItemGroup>
+
+  <!-- Only enable API Compat when needed for some building assembly -->
+  <PropertyGroup Condition="'$(RunNetFrameworkApiCompat)'==''">
+    <RunNetFrameworkApiCompat>false</RunNetFrameworkApiCompat>
+    <RunNetFrameworkApiCompat Condition="'$(Net48CompatNeededProjects)'!='' 
+                                          and $(Net48CompatNeededProjects.Contains('$(MSBuildProjectName)'))">true</RunNetFrameworkApiCompat>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(RunReferenceApiCompat)'==''">
+    <RunReferenceApiCompat>false</RunReferenceApiCompat>
+
+    <RefBin>$(ArtifactsDir)bin\$(AssemblyName)\$(Configuration)\$(Platform)\$(AssemblyName).dll</RefBin>
+
+    <!-- TODO:  Re-enable and fix matching. -->
+    <!-- Since some of our ref/lib pairs can potentially be built internally, only run when the paired lib exists. -->
+    <!--<RunReferenceApiCompat Condition="'$(ReferenceCompatNeededProjects)'!='' 
+                                      and $(ReferenceCompatNeededProjects.Contains('$(MSBuildProjectName)'))
+                                      and Exists('$(ArtifactsDir)bin\$(AssemblyName)\$(Configuration)\$(Platform)\$(AssemblyName).dll')">true</RunReferenceApiCompat>-->
+
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <RunApiCompat>false</RunApiCompat>
+    <RunApiCompat Condition="'$(RunNetFrameworkApiCompat)'=='true' or '$(RunReferenceApiCompat)'=='true'">true</RunApiCompat>
+
+    <RunApiCompatForSrc>false</RunApiCompatForSrc>
+    <RunApiCompatForSrc Condition="'$(RunNetFrameworkApiCompat)'=='true'">true</RunApiCompatForSrc>
+
+    <RunMatchingRefApiCompat>false</RunMatchingRefApiCompat>
+    <RunMatchingRefApiCompat Condition="'$(RunReferenceApiCompat)'=='true'">true</RunMatchingRefApiCompat>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <TaskFactory>RoslynCodeTaskFactory</TaskFactory>
```

Comment:
is this `TaskFactory` needed now that you don't have an inline task?

Reply:
Yeah I have another change already that will address it, I'll resolve when I push it.